content/en/docs/operator.md

```diff
@@ -0,0 +1,61 @@
+
+We use Operators because managing stateful applications, like databases, caches and monitoring systems, is a big challenge, especially at massive scale. These systems require human operational knowledge to correctly scale, upgrade and reconfigure while at the same time protecting against data loss and unavailability.
+
+All Operators use the controller pattern, but the converse is not true. It's only an Operator if it has API extension and single-app focus in addition to the controller pattern
+
+## What do Operators provide?
+
+- Repeatability of installation and upgrade.
+- Constant health checks of every system component.
+- Over-the-air (OTA) updates for Kubernetes components and ISV content.
+- A place to encapsulate knowledge from field engineers and spread it to all users, not just one or two.
+
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. It is part of the Operator Framework, an open source toolkit designed to manage Kubernetes native applications (Operators) in an effective, automated, and scalable way.
+
+
+## How does OLM enable cluster admins and Developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.X, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+
+For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring, and big data services without having to be subject matter experts, because the Operator has that knowledge baked into it.
+
+
+## Build with the Operator SDK
+
+The [Operator SDK](https://sdk.operatorframework.io/) provides the tools to build, test and package Operators. The Operator SDK strips away a lot of the boilerplate code that is normally required to integrate with the Kubernetes API. It also provides a usable scaffolding so developers can focus on adding business logic (for example, how to scale, upgrade, or backup the application it manages). Leading practices and code patterns shared across Operators are included in the Operator SDK to help prevent duplicating efforts. The Operator SDK also encourages short, iterative development and test cycles with tooling that allow for basic validation of the Operator, and automated packaging for deployment using the [Operator Lifecycle Manager](https://olm.operatorframework.io/).
```

Comment:
```suggestion
[Operator SDK](https://sdk.operatorframework.io/) is also a component of the [Operator Framework](https://github.com/operator-framework). Writing an operator today can be difficult because of challenges such as using low-level APIs, writing boilerplate, and a lack of modularity which leads to duplication.

The [Operator SDK](https://sdk.operatorframework.io/) is a framework that uses the [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) library to make writing operators easier by providing:

- High-level APIs and abstractions to write the operational logic more intuitively
- Tools for scaffolding and code generation to bootstrap a new project fast
- Extensions to cover common operator use cases

Note that [Operator SDK](https://sdk.operatorframework.io/) provides helpers and features to help you [integrate your project with OLM](https://sdk.operatorframework.io/docs/olm-integration/).  For further information check https://sdk.operatorframework.io/ .

```